amundsen_application/static/js/components/common/ResourceListItem/DashboardListItem/index.tsx

```diff
@@ -23,7 +23,11 @@ export interface DashboardListItemProps {
 class DashboardListItem extends React.Component<DashboardListItemProps, {}> {
   getLink = () => {
     const { dashboard, logging } = this.props;
-    return `/dashboard?uri=${dashboard.uri}&index=${logging.index}&source=${logging.source}`;
+    const { cluster, product, group_url, uri } = dashboard;
+    const groupID = group_url.split('/').pop();
+    const dashboardName = uri.split('/').pop();
+
+    return `/dashboard/${cluster}/${product}/${groupID}/${dashboardName}?index=${logging.index}&source=${logging.source}`;
```

Comment:
How did we come up with this specific path? If we use new concepts that the frontend has to parse and generate, then we end up introducing hardcoded url & uri processing logic that is specific to Mode. If community members are using some other tool, this parsing logic could pose issues for them unless all BI tools somehow format their values the same exact way. 

I would propose that we only use some combination of the exact values that come from the metadata service, and use `group_name` and `name` directly from our metadata.

Reply:
This was a suggestion from @danwom, happy to adapt it.

I will follow your suggestion and move the URL creation into a helper so it is easy to change in the future.